std/file.d

```diff
@@ -237,14 +237,14 @@ void[] read(R)(R name, size_t upTo = size_t.max)
     import std.utf : byChar;
     scope(exit)
     {
-        assert(exists("someUniqueFilename"));
-        remove("someUniqueFilename");
+        assert(exists(deleteme));
+        remove(deleteme);
```

Comment:
Should I do that for [all five tests that use someUniqueFilename and are in the docs](http://dlang.org/phobos/std_file.html#.readText) or only the first one?